netsim/devices/dellos10.py

```diff
@@ -3,31 +3,33 @@
 #
 from box import Box
 
-from . import _Quirks,need_ansible_collection
+from . import _Quirks,need_ansible_collection,report_quirk
 from ..utils import log
 from ..augment import devices
 
-def check_vlan_ospf(name: str, iflist: list, vname: str) -> None:
+def check_vlan_ospf(node: Box, vname: str) -> None:
+  name = node.name
+  iflist = node.get('interfaces',[])
```

Comment:
This is wrong - it should take into account the vrf, if not 'default'

Reply:
Indeed. Thank you!